### ISSUE TITLE
Create a release test to control the number of senders of #notEmpty

### ISSUE THREAD

**Issue by astares** [guillep/pharo-core#15125]
Currently we have 264 senders of #notEmpty which we should change in favour of using #isNotEmpty (see related issue #15120)
This is distributed to standard Pharo repo but also Spec, Iceberg repo and others

```Smalltalk
SystemNavigation default allSendersOf: #notEmpty
```

Lets create a release test to control the number of senders of #notEmpty so we do not increase the number but can bring the number
down step by step and then finally be able to solve #15120

![image](https://github.com/pharo-project/pharo/assets/5980033/82ecd1a7-2150-40b3-b430-c665fb205dc8)

**Comment by Ducasse** [comment]
Good catch!
May be could use the automatic deprecation to rewrite the callers?

**Comment by astares** [comment]
@Ducasse yes, we will use a transformation (as I described in #15118) and we will clean senders step by step

Did the same for [XMLParser ](https://github.com/pharo-contributions/XML-XMLParser) library (see https://github.com/pharo-contributions/XML-XMLParser/commit/2f63cbb2450643b7ca1d0b3fbd11969134dd54c3#diff-ce852f18c23e5fc685138b851f6d4151922ebac330a7962f259ece6724dda1c0R38) where I deprecated it too yesterday for the new release

![image](https://github.com/pharo-project/pharo/assets/5980033/b24c328f-70b1-4422-a686-91edabcbf445)

But before that I will try to clean senders to bring the number down
